app/controllers/courts_controller.rb

```diff
@@ -65,61 +48,15 @@ def show
       respond_to do |format|
         format.html
         format.json do
-          render content_type:'application/ld+json'
+          render content_type: 'application/ld+json'
         end
       end
     end
   end
 
   private
-  def find_court
-    @court = Court.friendly.find(params[:id])
-  end
 
-  def courts_csv
```

Comment:
@leoapost  Well, the tests are commented out. So I assume that it isn't. I can ask Dan just to be sure.